src/localization/translations/en.json

```diff
@@ -134,5 +134,10 @@
     "onboard_summary_title": "Your entered details",
     "onboard_summary_subtitle": "When you press on finalize, you accept the data you entered and you will proceed to the login screen. The personal data can be changed in the settings.",
     "onboard_summary_info_tab_header_label": "Your information",
-    "onboard_summary_button_caption": "Finalize and go to my wallet"
+    "onboard_summary_button_caption": "Finalize and go to my wallet",
+    "credentials_required_title": "Required credentials",
+    "credentials_required_subtitle": "The verifying party: {0} would like to receive the following information from you for verification.",
```

Comment:
`The verifying party:` can be removed in its entirety

Reply:
removed The verifying party: